pkg/plugins/golang/v3/init.go

```diff
@@ -171,12 +172,23 @@ func checkDir() error {
 			if strings.HasPrefix(info.Name(), ".") {
 				return nil
 			}
+			// Allow files ending with '.md' extension
+			if strings.HasSuffix(info.Name(), ".md") && !info.IsDir() {
+				return nil
+			}
+			// Allow capitalized files (like LICENSE, CODEOWNERS..)
```

Comment:
I'm not sure if we should allow capitalized files. That prevents us from using any other capitalized file in the future as the user could have already have one and it would break something that previously worked.

Reply:
As a personal feedback, I get stuck with multiple kubebuilder init because of this restriction.
They're more and more automations that create repo with this kind of files naming. 
Btw I understand your point too, as you want then.

Reply:
(IMO) The problem here is that the PROJECT file is capitalized. This one we could not allow have already.

Reply:
seems solved

Reply:
@Adirio status still "requested changes", you need me to modify smthg before approval?